docs/source/backends/vpn/wireguard.rst

```diff
@@ -0,0 +1,193 @@
+=================
```

Comment:
is this file listed in the toctree? I tried generating the docs and don't see it in the menu.

Reply:
@nemesisdesign can this be resolved?